spec/spec_helper.rb

```diff
@@ -18,6 +18,16 @@ def require_domain_file
 RSpec.configure do |config|
   config.include Spec::Support::AutomationHelper
 
+  config.around(:example) do |ex|
+    begin
+      ex.run
+    rescue SystemExit => e
+      puts "Kernel.exit called from:"
```

Comment:
Can you throw an extra STDERR.puts above the one saying "Kernel.exit".  I think this would make it easier to read, otherwise the "Kernel.exit" bit is right up against the row of dots.